MekHQ/src/mekhq/campaign/parts/CombatInformationCenter.java

```diff
@@ -0,0 +1,249 @@
+	    int time = 0;
+        if (campaign.getCampaignOptions().useAeroSystemHits()) {
+            //Test of proposed errata for repair times
+            time = 120;
+            if (unit != null && unit.getEntity().hasNavalC3()) {
+                time *= 2;
+            }
+            if (hits == 1) {
+                time *= 1;
+            } 
+            if (hits == 2) {
+                time *= 2;
+            }
+            return time;
+        }
+		if(isSalvaging()) {
+			time = 4320;
+		} else {
+		    time = 120;
+		}
+		return time;
+	}
+	
+	@Override
+	public int getDifficulty() {
+	    if (campaign.getCampaignOptions().useAeroSystemHits()) {
+            //Test of proposed errata for repair time and difficulty
+            if (hits == 1) {
+                return 1;
+            } 
+            if (hits == 2) {
+                return 2;
+            }
+        }
+		if(isSalvaging()) {
+			return 0;
+		}
+		return 1;
+	}
+
+	@Override
+	public void updateConditionFromPart() {
+		if(null != unit && unit.getEntity() instanceof Aero) {
+			((Aero)unit.getEntity()).setCICHits(hits);
+		}
+		
+	}
+
+	@Override
+	public void fix() {
+		super.fix();
+		if(null != unit && unit.getEntity() instanceof Aero) {
+			((Aero)unit.getEntity()).setCICHits(0);
+		}
+	}
+
+	@Override
+	public void remove(boolean salvage) {
+		if(null != unit && unit.getEntity() instanceof Aero) {
+			((Aero)unit.getEntity()).setCICHits(3);
+			Part spare = campaign.checkForExistingSparePart(this);
+			if(!salvage) {
+				campaign.removePart(this);
+			} else if(null != spare) {
+				spare.incrementQuantity();
+				campaign.removePart(this);
+			}
+			unit.removePart(this);
+			Part missing = getMissingPart();
+			unit.addPart(missing);
+			campaign.addPart(missing, 0);
+		}
+		setUnit(null);
+		updateConditionFromEntity(false);
+	}
+
+	@Override
+	public MissingPart getMissingPart() {
+		return new MissingCIC(getUnitTonnage(), cost, campaign);
+	}
+
+	@Override
+	public String checkFixable() {
+	    if (isSalvaging()) {
+            // FCS/CIC computers are designed for and built into the ship. Can't salvage and use somewhere else
+            return "You cannot salvage a spacecraft CIC. You must scrap it instead.";
+        }
+		return null;
+	}
+
+	@Override
+	public boolean needsFixing() {
+		return hits > 0;
+	}
+
+	@Override
+	public Money getStickerPrice() {
+		calculateCost();
+		return cost;
+	}
+
+	public void calculateCost() {
+		if(null != unit) {
+		    // There's more to CIC than just Fire Control
+		    // Use Bridge + Computer + FC Computer + Gunnery Control System costs, p158 SO.
+		    cost = Money.of(200000 + (10 * unit.getEntity().getWeight()) + 200000 + 100000 + (10000 * ((Jumpship)unit.getEntity()).getArcswGuns()));
+		}
+	}
+	
+	@Override
+	public double getTonnage() {
+		return 0;
+	}
+
+	@Override
+	public boolean isSamePartType(Part part) {
+		return part instanceof CombatInformationCenter && cost == part.getStickerPrice();
```

Comment:
`Money` is a reference type and needs to use `.equals()` instead of `==`. Also we need to make sure `calculateCost` is called first.
```suggestion
		return part instanceof CombatInformationCenter && getStickerPrice().equals(part.getStickerPrice());
```

Reply:
Fixed